SparkPost/support-docs
# EA-9267 - Add support and placeholders for IT+CT+DT Docs

- Give your PR a recognizable title. For example: "FE-123: Add new prop to component" or "Resolve Issue #123: Fix bug in component"
- Your PR title will be visible in changelogs

### What Changed

- added `component/site/base*.tsx` files
   - `baseHomePageContent.tsx` 
   -  `baseNavigation.tsx`
- added `component/site/analyst*.tsx` files
   - `analystHomePageContent.tsx`
   - `analystLayout.tsx`
   - `analystNavigation.tsx`
- Updated `component/site/doc*tsx` files to extend `base*tsx` files
   - Should look & function exactly same as now
- Added ability to hide drawer buttons (login, try now, dashboard - header buttons)
   - Only hidden on `/analyst` pages
- Updated build script to include building algolia for new `next_analyst_documentation`

![image](https://user-images.githubusercontent.com/5553048/221048505-99861465-cdfa-45f1-b931-89761f36d81f.png)

![image](https://user-images.githubusercontent.com/5553048/221048624-c1ccc45b-6650-4145-b8e3-c9218edd4477.png)


### How To Test or Verify

- start up app
- check that `localhost:3000/analyst` functions

### PR Checklist

Below are some checklists to follow for the correct procedure in different circumstance. The first list ("All PRs Checklist") should be followed for ALL PRs. The next 2 are additive to this list depending on what type of PR you are using.

For example: If you are submitting a content change to one of the support documents, your checklist would include the:

- "All PRs Checklist"
- AND the "Content Changes Checklist

If you are submitting a feature addition, enhancement, or bug fix, your checklist would include the:

- "All PRs Checklist"
- AND the "Development Changes Checklist"

#### All PRs Checklist


#### Content Changes Checklist


#### Development Changes Checklist (some checks are automatic github actions and will not be listed here. ie. "all tests pass")
